src/components/Chakra/Stack.stories.mdx

```diff
@@ -0,0 +1,129 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+import { Box, Stack, HStack, VStack, Circle } from "@chakra-ui/react";
+
+import { getCategory } from "../../utils/componentCategories";
+import Icon from "../Icons/Icon";
+import { IconNames, IconRotationTypes, IconColors } from "../Icons/IconTypes";
+import Radio, { onChangeDefault } from "../Radio/Radio";
+
+<Meta title={getCategory("Stack, HStack, VStack")} component={Stack} />
+
+# Stack, HStack, VStack
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.24.0`   |
+
+Note: This needs the use of the `DSProvider` component. See the README for more information.
+
+These components are directly exported from Chakra UI. The `Stack`, `HStack`,
+and `VStack` are layout component that makes it easy to stack elements together
+with spacing between them.
+
+All available props can be found on the [Style Props](https://chakra-ui.com/docs/features/style-props)
+guide on the Chakra UI site.
+
+## Stack
+
+The `Stack` component is the most flexible component because it can stack elements
+both horizontally and vertically through the `direction` prop.
+
+<Canvas>
+  <Story
+    name="Stack, HStack, VStack"
+    args={{
+      direction: "column",
+      spacing: "10px",
+    }}
+    argTypes={{
+      direction: {
+        control: {
+          type: "select",
+        },
+        options: ["column", "row"],
+      },
+    }}
+  >
+    {(args) => (
+      <Stack {...args}>
+        <Circle size="50px" borderColor="ui.black" borderWidth="1px">
+          <Icon name={IconNames.error_filled} />
+        </Circle>
+        <Circle size="50px" borderColor="ui.black" borderWidth="1px">
+          <Icon name={IconNames.error_outline} />
+        </Circle>
+        <Circle size="50px" borderColor="ui.black" borderWidth="1px">
+          <Icon name={IconNames.check} />
+        </Circle>
+      </Stack>
+    )}
+  </Story>
+</Canvas>
+
+## HStack
+
+Stacks components horizontally.
+
+<Canvas>
+  <HStack>
+    <Radio
+      checked
+      labelText="Tom Nook"
+      name="testChecked"
+      onChange={onChangeDefault}
+      showLabel={true}
+    />
+    <Radio
+      labelText="Timmy"
+      name="testChecked"
+      onChange={onChangeDefault}
+      showLabel={true}
+    />
+    <Radio
+      labelText="Tommy"
+      name="testChecked"
+      onChange={onChangeDefault}
+      showLabel={true}
+    />
+    <Radio
+      labelText="Celeste"
+      name="testChecked"
+      onChange={onChangeDefault}
+      showLabel={true}
+    />
+  </HStack>
+</Canvas>
+
+## VStack
```

Comment:
Just a comment: this example shows why `Radio` and `Checkbox` need a parent wrapper.